Source/OpenTK/Platform/Windows/WinInputBase.cs

```diff
@@ -58,7 +58,9 @@ public WinInputBase()
             WndProc = WindowProcedure;
 
             InputThread = new Thread(ProcessEvents);
+#if !_NET_CORECLR
             InputThread.SetApartmentState(ApartmentState.STA);
```

Comment:
I wonder if this is problematic at all? I believe that STA is the default, though, and since we don't expose any way to set this, I guess it shouldn't be an issue.